### PR TITLE
Lf 1240/resizeable input

### PR DESCRIPTION
**Pull request by audreyylu**

### CODE REVIEW

File: packages/webapp/src/components/LocationDetailLayout/PointDetails/index.js [hunk 1]
```diff
@@ -57,14 +58,15 @@ export default function PointDetailsLayout({
...
-      />
+      <InputAutoSize optional={true} label={t('common:NOTES')} style={{ marginBottom: '40px'}} hookFormRegister={register(pointEnum.notes)} disabled={isViewLocationPage}/>
+      {/*<Input*/}
+      {/*  label={t('common:NOTES')}*/}
+      {/*  type="text"*/}
+      {/*  optional*/}
+      {/*  style={{ marginBottom: '40px' }}*/}
+      {/*  hookFormRegister={register(pointEnum.notes)}*/}
+      {/*  disabled={isViewLocationPage}*/}
+      {/*/>*/}
```
Comment (beetai) [inline packages/webapp/src/components/LocationDetailLayout/PointDetails/index.js:69]: Remove commented code, and `InputAutoSize` space props across multiple lines

File: packages/webapp/src/stories/Form/InputAutoSize/InputAutoSize.stories.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+export const withError = Template.bind({});
+withError.args = {
+  label: 'Notes',
+  defaultValue: 'default value',
+  rowsMax: 4,
+  optional: true,
+  errors: 'error',
+  disabled: false,
+
+}
```
Comment (beetai) [inline packages/webapp/src/stories/Form/InputAutoSize/InputAutoSize.stories.js:38]: eol whitespace pls

File: packages/webapp/src/components/LocationDetailLayout/PointDetails/index.js (hunk 1 above)
Comment (beetai) [reply packages/webapp/src/components/LocationDetailLayout/PointDetails/index.js:69]: Should do this for `LocationDetailLayout/AreaDetails/index.js` and `LocationDetailLayout/LineDetails/index.js` as well

File: packages/webapp/src/components/Form/InputAutoSize/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+.textArea:focus:enabled {
+  outline: none;
+  border-color: var(--inputActive);
+}
+
+.textArea:disabled {
+  background-color: var(--inputDisabled) !important;
+  color: var(--grey600);
+  border-color: var(--inputDefault);
+}
```
Comment (beetai) [inline packages/webapp/src/components/Form/InputAutoSize/styles.module.scss:41]: eol whitespace

File: packages/webapp/src/components/Form/InputAutoSize/index.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  onChange,
+  errors,
+  ...props
+}) {
+  const input = useRef();
+  const name = hookFormRegister?.name ?? props?.name;
+  const { t } = useTranslation(['translation', 'common']);
+
+
+  console.log(clsx(styles.textArea, errors && styles.inputError));
```
Comment (beetai) [inline packages/webapp/src/components/Form/InputAutoSize/index.js:28]: remove console log

**Review by beetai — CHANGES_REQUESTED** [review]
Good to go after these minor changes. Good work

**Review by beetai — APPROVED** [review]

File: packages/webapp/src/components/LocationDetailLayout/AreaDetails/index.js [hunk 5]
```diff
@@ -110,14 +111,11 @@ export default function AreaDetails({
...
-        label={t('common:NOTES')}
-        type="text"
-        optional
-        hookFormRegister={register(areaEnum.notes)}
-        style={{ marginBottom: '40px' }}
-        disabled={isViewLocationPage}
-      />
+      <InputAutoSize optional={true}
+                     label={t('common:NOTES')}
+                     tyle={{ marginBottom: '40px'}}
```
Comment (JimmyRowland) [inline packages/webapp/src/components/LocationDetailLayout/AreaDetails/index.js:116]: tyle => style

File: packages/webapp/src/stories/Form/InputAutoSize/InputAutoSize.stories.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  optional: true,
+};
+
+export const Empty = Template.bind({});
+Empty.args = {
+  label: 'Notes',
+  rowsMax: 4,
+  optional: true,
+};
+
```
Comment (JimmyRowland) [inline packages/webapp/src/stories/Form/InputAutoSize/InputAutoSize.stories.js:28]: add
export const Disabled = Template.bind({});
Disabled.args = {
  label: 'disabled',
  rowsMax: 4,
  optional: true,
  disabled: true
};

File: packages/webapp/src/stories/Form/InputAutoSize/InputAutoSize.stories.js [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+  component: InputAutoSize,
+  decorators: componentDecorators,
+};
+
+const Template = (args) => <InputAutoSize {...args} />;
+
+
+export const Primary = Template.bind({});
+Primary.args = {
+  label: 'Notes',
```
Comment (JimmyRowland) [inline packages/webapp/src/stories/Form/InputAutoSize/InputAutoSize.stories.js:16]: label: 'Optional'

File: packages/webapp/src/stories/Form/InputAutoSize/InputAutoSize.stories.js [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+export default {
+  title: 'Components/InputAutoSize',
+  component: InputAutoSize,
+  decorators: componentDecorators,
+};
+
+const Template = (args) => <InputAutoSize {...args} />;
+
+
+export const Primary = Template.bind({});
```
Comment (JimmyRowland) [inline packages/webapp/src/stories/Form/InputAutoSize/InputAutoSize.stories.js:14]: export const Optional
Optional.args={...

File: packages/webapp/src/stories/Form/InputAutoSize/InputAutoSize.stories.js [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+  rowsMax: 4,
+  optional: true,
+};
+
+export const withError = Template.bind({});
+withError.args = {
+  label: 'Notes',
+  defaultValue: 'default value',
+  rowsMax: 4,
+  optional: true,
```
Comment (JimmyRowland) [inline packages/webapp/src/stories/Form/InputAutoSize/InputAutoSize.stories.js:34]: remove optional

File: packages/webapp/src/stories/Form/InputAutoSize/InputAutoSize.stories.js [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+
+export const Empty = Template.bind({});
+Empty.args = {
+  label: 'Notes',
+  rowsMax: 4,
+  optional: true,
+};
+
+export const withError = Template.bind({});
+withError.args = {
```
Comment (JimmyRowland) [inline packages/webapp/src/stories/Form/InputAutoSize/InputAutoSize.stories.js:30]: WithError

File: packages/webapp/src/stories/Form/InputAutoSize/InputAutoSize.stories.js [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+};
+
+export const Empty = Template.bind({});
+Empty.args = {
+  label: 'Notes',
+  rowsMax: 4,
+  optional: true,
+};
+
+export const withError = Template.bind({});
```
Comment (JimmyRowland) [inline packages/webapp/src/stories/Form/InputAutoSize/InputAutoSize.stories.js:29]: WithError

**Review by JimmyRowland — CHANGES_REQUESTED** [review]

**Review by JimmyRowland — APPROVED** [review]

**Pull request merged**